### PR TITLE
Fsm garima

### PR DESCRIPTION
**Pull request by chinmoy-eGov**

### CODE REVIEW

File: web/micro-ui-internals/packages/libraries/src/services/molecules/FSM/GarimaSearchActions.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { FSMService } from "../../elements/FSM";
+
+const GarimaSearchActions = async (garimaData, tenantId) => {
+  garimaData = {"garimaId":garimaData};
+  try {
+    const response = await FSMService.searchGarimaDetails({...garimaData}, tenantId);
+    return response?.data;
+  } catch (error) {
+    // throw new Error(error?.response?.data?.Errors[0].message);
```
Comment (anilsingha-eGov) [inline web/micro-ui-internals/packages/libraries/src/services/molecules/FSM/GarimaSearchActions.js]: log error properly @chinmoy-eGov

File: web/micro-ui-internals/packages/modules/fsm/src/pageComponents/GarimaPersonalDetails.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+
+const GarimaPersonalDetails = ({ config, onSelect, formData = {}, userType, id }) => {
+  const tenantId = Digit.ULBService.getCurrentTenantId();
+  const mobileView = Digit.Utils.browser.isMobile() ? true : false;
+  const { t } = useTranslation();
+  const [showToast, setShowToast] = useState(null);
+
+  const formConfig = [
```
Comment (anilsingha-eGov) [inline web/micro-ui-internals/packages/modules/fsm/src/pageComponents/GarimaPersonalDetails.js]: create  a new  config file  @chinmoy-eGov and import here

File: web/micro-ui-internals/packages/modules/fsm/src/pageComponents/SelectvehicleCapacity.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  const { pathname: url } = useLocation();
+  const editScreen = url.includes("/modify-application/");
+
+  // const { data: channelMenu } = Digit.Hooks.fsm.useMDMS(tenantId, "FSM", "EmployeeApplicationChannel");
+  // const [channel, setChannel] = useState(formData?.channel);
+
+  // function selectChannel(value) {
+  //   setChannel(value);
+  //   onSelect(config.key, value);
+  // }
```
Comment (anilsingha-eGov) [inline web/micro-ui-internals/packages/modules/fsm/src/pageComponents/SelectvehicleCapacity.js]: Remove commented code @chinmoy-eGov

File: web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+    
+
+    // const {
+    //     isLoading: isDriverLoading,
+    //     isError: isDriverError,
+    //     data: driverData,
+    //     error: driverError,
+    //     mutate,
+    // } = Digit.Hooks.fsm.useGarimaSearchActions(tenantId);
```
Comment (anilsingha-eGov) [inline web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js]: Remove  this one also @chinmoy-eGov

File: web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+    //       },
+    //       onSuccess: (data, variables) => {
+    //         setGarimaDriverData (data)
+    //         let garimaDriverDetails = formData.garimaDriverDetails || {};
+    //         garimaDriverDetails.garimaDriverMobileNumber = data[0].mobile_number;
+    //         garimaDriverDetails.garimaDriverName =  `P${data[0].garima_id}`;
+    //         onSelect(configKey, { ...formData[configKey], garimaDriverDetails: garimaDriverDetails });
+    //       },
+    //     });
+    // };
```
Comment (anilsingha-eGov) [inline web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js]: Remove  this one also @chinmoy-eGov

File: web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+    const configs = [
```
Comment (anilsingha-eGov) [inline web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js:77]: Create new config file for this and import @chinmoy-eGov

**Review by anilsingha-eGov — CHANGES_REQUESTED** [review]

File: web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js (hunk 6 above)
Comment (chinmoy-eGov) [reply web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js:77]: We are sending hook data through config, so we can not create a new config for this case

Comment (anilsingha-eGov) [reply web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js:77]: ok

**Review by anilsingha-eGov — APPROVED** [review]

**Pull request merged**